spring/spring-boot-session-replication-spring-session-hazelcast/src/main/java/com/hazelcast/guide/controller/SessionController.java

```diff
@@ -92,18 +92,27 @@ public String getSessionInfo(HttpServletRequest request) {
     }
 
     private String toHtmlTable(Map<String, Object> attributes) {
-        StringBuilder html = new StringBuilder("<html>");
-        html.append("<table border=\"1\" cellpadding=\"5\" cellspacing=\"5\">");
-        attributes.forEach((k, v) -> addHtmlTableRow(html, k, v));
-        html.append("</table></html>");
-        return html.toString();
+        String html = """
+                <html>
+                    <body>
+                        <table style="border-spacing: 5px; border: 1px solid black;">
+                        %s
+                        </table>
+                    </body>
+                </html>
+                """;
+        StringBuilder rows = new StringBuilder("<html><body>");
+        attributes.forEach((k, v) -> rows.append(addHtmlTableRow(k, v)));
```

Comment:
```suggestion
        String rows = attributes.entrySet()
                .stream()
                .map(entry -> addHtmlTableRow(entry.getKey(), entry.getValue()))
                .collect(Collectors.joining());
```

nit - you can avoid a `StringBuilder`.

Not a nit - I think your adding `<html><body>` twice.

Reply:
That's a good idea! cc9a0caecbba266fe5423143524873dc3b50b60a